engines/wasmer/example_test.go

```diff
@@ -0,0 +1,30 @@
+package wasmer
+
+import (
+	"context"
+	"log"
+
+	"github.com/wasmerio/wasmer-go/wasmer"
+
+	"github.com/wapc/wapc-go"
+)
+
+// This shows how to customize the underlying wasmer engine used by waPC.
+func Example_custom() {
+	// Set up the context used to instantiate the engine.
+	ctx := context.Background()
+
+	// Configure waPC to use a specific wasmer feature.
+	e := EngineWithRuntime(func() (*wasmer.Engine, error) {
```

Comment:
someone may feel like this should be `EngineWithEngine` to match the type, but I feel runtime is nicer than stuttering.

Reply:
Agreed.  I was thinking along those lines as well, but the name is cumbersome to say the least.